### PR TITLE
ZCS-6140:GraphQL API for CreateMountpoint, SendShareNotification

### PR DESCRIPTION
**Pull request by shrikant-prasad**
GraphQL API for CreateMountpoint, SendShareNotification and GetShareInfo

### CODE REVIEW

File: soap/src/java/com/zimbra/soap/mail/type/Mountpoint.java [hunk 1]
```diff
@@ -124,6 +134,7 @@ public void setReminderEnabled(Boolean reminderEnabled) {
         this.reminderEnabled = ZmBoolean.fromBool(reminderEnabled);
     }
 
+    @GraphQLQuery(name=GqlConstants.BROKEN, description="If \"tr\" is true in the request, broken is set if this is a broken link")
```
Comment (desouzas) [inline soap/src/java/com/zimbra/soap/mail/type/Mountpoint.java]: `isBroken` ?

File: soap/src/java/com/zimbra/soap/type/ShareInfo.java [hunk 2]
```diff
@@ -128,72 +133,85 @@ public ShareInfo() {
 
     public void setOwnerId(String ownerId) { this.ownerId = ownerId; }
 
+    @GraphQLQuery(name=GqlConstants.OWNER_ID, description="owner id")
```
Comment (desouzas) [inline soap/src/java/com/zimbra/soap/type/ShareInfo.java]: Add `@GraphQLNonNull` above each query annotation for all of the following except mountpoint id.

Comment (shrikant-prasad) [reply soap/src/java/com/zimbra/soap/type/ShareInfo.java]: added

File: soap/src/java/com/zimbra/soap/mail/type/Mountpoint.java (hunk 1 above)
Comment (shrikant-prasad) [reply soap/src/java/com/zimbra/soap/mail/type/Mountpoint.java]: value of BROKEN was is Broken, now changed var name also.

**Pull request closed without merging**